plugin.js

```diff
@@ -24,6 +24,7 @@ const plugin = async (fastify, config) => {
     await pool.close()
   })
   fastify.decorate('mssql', { pool })
+  fastify.decorate('sqlTypes', MSSql)
```

Comment:
No strong opinion here but there are a couple of things that I don't like much about this approach:

- one plugin which exposes 2 decorators
- a decorator called `sqlTypes` which in fact contains the whole mssql library

I'm not sure if it would be really better but have you considered keeping only the existing decorator and just merge our custom `pool` object into the mssql default export instead? So the user can access everything from a single place

Reply:
I did not think about this option: good suggestion!

Done

Reply:
Since mssql is based on Tedious, this is what we currently do in Tedious.js to bring the Types, so I am hoping your solution will be similar.   Maybe someting like mssql.TYPES.Int.
```
const TYPES = require('tedious').TYPES;
sqlreq.addParameter('userID',TYPES.Int,userID);
```

Reply:
We're doing something similar in this PR already, but since we're encapsulating mssql in a fastify decorator, we decided to export the types from there. Have a look at the changes to see what it looks like. would that work for you?